README.md

```diff
@@ -55,11 +56,25 @@ The following types of data are being considered:
  | restricted  | Redistribution only with permission|
  | public      | No redistribution limits beyond licenses|
 
-4. Custom labels may be defined by a namespace prefix using reverse DNS notation of a domain controlled by the author. For example, `com.redhat.access`.
+1. Possible values of config field. Application configuration may be defined or hinted using the config label. This allows documentation and tooling to provide parameterization control. Currently, environment variables are the only parameter, with three attributes.
+
+ |Name         | Description |
+ |-------------|-------------|
+ | config.env.KEY             | Default value. Use empty string "" if no default.|
```

Comment:
i think we were doing to skip having a "default" value label and just try to pick up the default from any ENV statement in the dockerfile.  But after we got off the phone, i had second thoughts and i think we still want to have an (optional) default label, but the name should be:
config.env.KEY.default

we were also going to add another optional label:
config.env.KEY.displayName

which is a value used to label the input field for the value in UIs.  (if not provided, the label for the field will just be "KEY")